app/Helpers/DatabaseHelper.php

```diff
@@ -64,22 +64,25 @@ public static function getResourcePathArray($id){
         return $res;
     }
 
-    public static function getObyxPoints($reason){
+    public static function getObyxPoints($reason) {
```

Comment:
__[code_sniffer]__
* Opening brace should be on a new line

<!-- Comment by [SideCI](https://sideci.com) -->